manifests/g/Grisbi/Grisbi/2.0.5/Grisbi.Grisbi.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Grisbi.Grisbi
+PackageVersion: 2.0.5
+PackageLocale: en-US
+Publisher: Grisbi
```

```suggestion
Publisher: Open Source
```
This must match Control Panel